packages/contracts/test/Sphinx.t.sol

```diff
@@ -44,4 +44,16 @@ contract Sphinx_Test is Test, Sphinx, SphinxTestUtils {
 
         assertEq(expectedAddress, sphinxModule);
     }
+
+    function test_sphinx_validate_reverts_empty_config() external {
+        sphinxConfig.owners = new address[](0);
+        sphinxConfig.threshold = 0;
+        sphinxConfig.testnets = new Network[](0);
+        sphinxConfig.mainnets = new Network[](0);
+        sphinxConfig.projectName = "";
+        vm.expectRevert(
+            "Sphinx: Detected an empty 'sphinxConfig' struct. Did you forget to add fields to it in your script's setUp function or constructor? If you've already added fields to it in your setUp function, have you called 'super.setUp()' in any contracts that inherit from your script?"
+        );
+        sphinxConfigABIEncoded();
+    }
```

Comment:
nit: The convention w/ our Forge test function names is generally in the format `test_<FUNCTION_NAME>_revert_<REASON>`